telemetry/sql/000008_waku_push_filter.up.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE IF NOT EXISTS wakuPushFilter (
+	id SERIAL PRIMARY KEY,
+	walletAddress VARCHAR(255),
+	peerIdSender VARCHAR(255) NOT NULL,
+	peerIdReporter VARCHAR(255) NOT NULL,
```

Comment:
what is `reporter` in this context? is it the node that we sent message to?

Reply:
Reporter is the node being run by the same user who sends this data to telemetry. When sending a message via lightpush, reporter and sender will be the same.